packages/docusaurus-theme-classic/src/theme/hooks/useTheme.js

```diff
@@ -20,6 +25,10 @@ const useTheme = () => {
 
   React.useEffect(() => {
     try {
+      if (disableDarkMode) {
```

Comment:
I'm not sure whether it's better to just do `return [null, () =>]` at the end instead, making guaranteeing that this hook doesn't do anything.

Reply:
LGTM, but why not put the condition __before__ `try` block?